.storybook/themes/commerce.scss

```diff
@@ -882,14 +882,16 @@
 					gap: var(--global-spacing-5),
 					margin: auto,
 					max-width: calc(var(--content-max-width) * 1px),
-					width: var(--content-scale-width),
+					width: 100%,
 				),
 				right-rail-main-right-rail: (
 					align-items: stretch,
 					grid-column: 1,
+					max-width: calc(var(--content-max-width) * 1px),
 				),
 				right-rail-main-interior-item: (
 					grid-column: 1,
+					max-width: calc(var(--content-max-width) * 1px),
```

Comment:
These just set the `max-width` to `1216px`.  The issue is in mobile where the content touches the edge of the screen and it should not except for the gallery and product image.  If the user places the anything in the `main` group and it is in the mobile viewport size (48rem / 768px), this doesn't actually restrict the width at all.

What we need to do is set the `max-width` of the commerce blocks (`product-information` and `product-content`) to the scale-width, with auto margins to center it and maximum width like `carousel-track`, `category-carousel-title`, etc.

Reply:
![image](https://user-images.githubusercontent.com/2287238/193324853-9cc347fd-fd85-4163-a77f-56d844d5b776.png)
![image](https://user-images.githubusercontent.com/2287238/193325208-96a0d68c-a176-4424-86aa-b4e3252d87b8.png)

Reply:
This way anyone can place the content in any of the panels (if they wanted the information to be on the left or right panel) and it would restrict itself from touching the edge while allowing the other components to fully stretch to the edge of the screen.